internal/cmd/secrets-manager/secrets_manager.go

```diff
@@ -0,0 +1,25 @@
+package secretsmanager
+
+import (
+	"github.com/stackitcloud/stackit-cli/internal/cmd/secrets-manager/instance"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/utils"
+
+	"github.com/spf13/cobra"
+)
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "secrets-manager",
+		Short: "Provides functionality for the Secrets Manager",
+		Long:  "Provides functionality for the Secrets Manager.",
```

Comment:
I think we can omit the `the`:

```suggestion
		Short: "Provides functionality for Secrets Manager",
		Long:  "Provides functionality for Secrets Manager.",
```